GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -231,6 +462,8 @@ gdjs.HowlerSoundManager.prototype._storeSoundInArray = function (arr, sound) {
   var index = null;
   for (var i = 0, len = arr.length; i < len; ++i) {
     if (arr[i] !== null && arr[i].canBeDestroyed()) {
+      //FIXME
+      //@ts-ignore
       arr[index] = sound;
```

Comment:
I am confused as to how this is working, the sound is always being set to index null? Isn't it meant to set `arr[i]` instead?

Reply:
Good catch! This is indeed a mistake. This means that the array of "free" sounds/musics will have a sound at index `null` when we try to replace a sound that was finished by another :/
The impact of this bug are likely that:
- As soon as there is one sound finished, all the new sounds are added to the index `null` (erasing the old gdjs.HowlerSound).
- this is kind of ok because as they are "free" sounds, we can't manipulate them using events (we can just launch them). Even if we override a `gdjs.HowlerSound` by another, the browser audio resource will still finish to play.
- ...but this means that when the app goes to background for example on Cordova, we try to pause all the sounds (both those on channels and these "free" sounds). We'll likely miss some.

This means that if I launch a lot of sound like explosions, then put my game in background, the explosions will "finish" to play.
I'm unsure if this means we're also leaking audio resources. In all cases, better fix this!